crates/testing/tests/tests_1/quorum_proposal_task.rs

```diff
@@ -184,6 +184,7 @@ async fn test_quorum_proposal_task_quorum_proposal_view_gt_1() {
     run_test_script(script, quorum_proposal_task_state).await;
 }
 
+#[ignore]
```

Comment:
Seems like breaking these will make the dependency task refactor more difficult. Perhaps I can take a look at these before this merges and try to get them working (minimally)?

Reply:
+1 I can also help with this.

Reply:
Btw `test_quorum_proposal_task_quorum_proposal_view_1` and `test_quorum_proposal_task_quorum_proposal_view_gt_1` aren't ignored but also failed, so we need to fix them as well.

Reply:
I'll talk to Brendon today about getting these running.

Reply:
Wait a second: https://github.com/EspressoSystems/HotShot/blob/main/crates/testing/tests/tests_1/quorum_proposal_task.rs.

I think this PR has a merge issue. I'm not sure how my gates got removed.

Reply:
As synced up offline, this test should have a `dependency_tasks` feature gate. But that's probably not the only issue, since other tests with this feature also fail.